discord_slash/client.py

```diff
@@ -560,6 +564,8 @@ def add_subcommand(self,
             "sub_group_desc": subcommand_group_description,
             "guild_ids": guild_ids,
             "api_options": options,
+            "default_permission": base_default_permission,
+            "api_permissions": base_permissions,
```

Comment:
The reason I needed is as `SubcommandObject` inherits `CommandObject` which means it will throw an error is those 2 fields are not present. .But thinking of it, I can just pass None value.

Reply:
Maybe it shouldn't inherit from CommandObject. :thinking: 
cc @eunwoo1104

Reply:
93b8865

Reply:
Honestly don't like this. It implies that the permissions do exist for subcommands but are always empty, which isn't the case. 
How about instead creating a base command class which then the normal command class inherits, and subcommands inherent?

Reply:
Sounds like a better option, just that it will have breaking change since SubcommandObject now inherits a different class.

Reply:
7e16da9

I made `CommandObject` the common class which `BasecommandObject` and `SubcommandObject` inherits from.

Reply:
Can this be resolved so we can merge this?

Reply:
oh its was hidden GitHub website, too many commits and comments HAHA, should be resolved already.